_includes/footer-phone.html

```diff
@@ -5,7 +5,7 @@
 <div class="footer-phone {{ small }} phone__row">
   
   <div class="footer-phone__column-left">
-    <img src="{{relroot}}img/phone-icon.svg" alt="">
+    <img src="{{relroot}}img/phone-icon.svg" alt="Telefoonicoon">
```

Comment:
Dit icoon is puur decoratief en heeft daarom bewust een leeg `alt` attribuut.

Reply:
Voor geinteresseerden (die dit later terug lezen en denken "ok maar waarom"): https://www.w3.org/WAI/tutorials/images/decorative/#image-used-as-part-of-page-design